### PR TITLE
Removed primary key for TeamName in tbl_Teams and fixed insert statem…

### PR DESCRIPTION
**Pull request by who-sane**
…ents. Also corrected connection string for DB.

### CODE REVIEW

File: nba-api-dotnet/appsettings.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DanDesktopDB": "Data Source=DESKTOP-CQVULK8;Initial Catalog=NBA;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
-    "AzureDatabase": "Server=tcp:nba-instance.database.windows.net,1433;Initial Catalog=nba-instance;Persist Security Info=False;User ID=nbaAdmin;Password=nba_prediction1;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
+    "AzureDatabase": "Server=tcp:nba-instance.database.windows.net,1433;Initial Catalog=NBA;Persist Security Info=False;User ID=nbaAdmin;Password=nba_prediction1;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
```
Comment (dan933) [inline nba-api-dotnet/appsettings.json:5]: Nice job good

File: nba-database/nbaProjectDBS.sql [hunk 2]
```diff
@@ -55,601 +54,601 @@ PRIMARY KEY (UserID)
...
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1630192,'Zeke','Nnaji',24,16,30,12,0.6,265,145,15,16,13,82,24,22);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1630533,'Ziaire','Williams',40,18,0,0,0.69,443,120,60,31,12,208,10,38);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1629597,'Zylan','Cheatham',0,1,0,0,0,0,0,0,0,0,3,0,0);
+INSERT INTO tbl_Players(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (203932,'Aaron','Gordon',44,28,29,21,0.611,1062,413,181,43,41,383,56,126);
```
Comment (dan933) [inline nba-database/nbaProjectDBS.sql:57]: Great good use of capitalisation and using s at the end.

**Review by dan933 — APPROVED** [review]
Nice well done on the bug fixes.

**Review by dan933 — APPROVED** [review]

File: nba-database/nbaProjectDBS.sql (hunk 2 above)
Comment (who-sane) [reply nba-database/nbaProjectDBS.sql:57]: thank.

**Pull request merged**